From: =?utf-8?q?Rog=C3=A9rio_Brito?= <rbrito@ime.usp.br>
Date: Wed, 21 Jun 2017 20:39:54 -0300
Subject: bin/kotlinc: patch upstream wrapper script to put jars into
 usr/share/java

---
 bin/kotlinc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/bin/kotlinc b/bin/kotlinc
index 9eacfbb..63b1db7 100755
--- a/bin/kotlinc
+++ b/bin/kotlinc
@@ -63,11 +63,11 @@ declare -a kotlin_app
 if [ -n "$KOTLIN_RUNNER" ];
 then
     java_args=("${java_args[@]}" "-Dkotlin.home=${KOTLIN_HOME}")
-    kotlin_app=("${KOTLIN_HOME}/lib/kotlin-runner.jar" "org.jetbrains.kotlin.runner.Main")
+    kotlin_app=("${KOTLIN_HOME}/share/java/kotlin-runner.jar" "org.jetbrains.kotlin.runner.Main")
 else
     [ -n "$KOTLIN_COMPILER" ] || KOTLIN_COMPILER=org.jetbrains.kotlin.cli.jvm.K2JVMCompiler
     java_args=("${java_args[@]}" "-noverify")
-    kotlin_app=("${KOTLIN_HOME}/lib/kotlin-preloader.jar" "org.jetbrains.kotlin.preloading.Preloader" "-cp" "${KOTLIN_HOME}/lib/kotlin-compiler.jar" $KOTLIN_COMPILER)
+    kotlin_app=("${KOTLIN_HOME}/share/java/kotlin-preloader.jar" "org.jetbrains.kotlin.preloading.Preloader" "-cp" "${KOTLIN_HOME}/share/java/kotlin-compiler.jar" $KOTLIN_COMPILER)
 fi
 
 "${JAVACMD:=java}" $JAVA_OPTS "${java_args[@]}" -cp "${kotlin_app[@]}" "${kotlin_args[@]}"
